modules/nw-rosa-proxy-remove-cli.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * networking/enable-cluster-wide-proxy.adoc
+
+:_content-type: PROCEDURE
+[id="nw-rosa-proxy-remove-cli_{context}"]
+= Removing the cluster-wide proxy using CLI
+
+You must use the `rosa` CLI to remove the proxy's address from your cluster.
+
+.Prerequisites
+
+* Cluster administrator permissions
+* {product-title} `rosa` CLI tool installed
+
+.Procedure
+
+. Use the `rosa edit` command to modify the proxy. You must pass empty strings to the `--http-proxy` and `--https-proxy` arguments to clear the proxy from the cluster:
++
+[source,terminal]
+----
+$ rosa edit cluster -c <cluster_name> --http-proxy "" --https-proxy "" 
+----
++
+[NOTE]
+====
+While your proxy may only use one of the proxy arguments, the empty field is ignored, so passing empty strings to both the `--http-proxy` and `--https-proxy` does not cause any issues.
+====
+
+
+. You receive the following response:
++
+.Sample Output
```

Comment:
This doesn't seem like a step, more like just the output of the previous step? If so, consider attaching it with a `+` after the previous `[NOTE]` (minus the sentence), and use "Example output" (per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output)):

```
+
.Example output
[source,yaml]
----
I: Updated cluster <cluster_name>
----
```

At that point, it would also become a single-step procedure, so would need to change the first step's . to a *, as well.